Tools/PowershellModule/src/Library/Remove-WinGetManifest.ps1

```diff
@@ -29,15 +26,11 @@ Function Remove-WinGetManifest
     [Optional] The Subscription name contains the Windows Package Manager rest source
```

Comment:
We should really just do a global REST replace across the entire script subfolders. somehow that change got lost?  we need to be careful with the automated replacement though